test/JDBC/upgrade/13_4/schedule

```diff
@@ -222,5 +222,6 @@ cast_eliminate
 TestDatatypeAggSort
 babel_index_nulls_order-before-15-5
 BABEL-2999
+drop_index-before-15_6-or-16_1
```

Comment:
Is this feature going in 16.1? I think it is meant to go in 16.2 and 15.6 so in that case the name should be `*-before-15_6-or-16_2`

Reply:
I understood 16_1 but let me know if not

Reply:
If its going in 16_1, then it should be *-before-15_5-or-16_1. If its going in 16_2, then it should be like what @rishabhtanwar29  suggests.

Reply:
I don't see it in 16_1 but I have included it in 16_2 today. Moreover, 16.1 seems to a standalone release and actual match for 15.6 is 16.2 so let's keep it in 16.2.

Reply:
Will correct

Reply:
So is it 16.1 or 16.2?